internal/impl/packages.go

```diff
@@ -183,11 +179,7 @@ func (d *Devbox) Remove(ctx context.Context, pkgs ...string) error {
 		return err
 	}
 
-	if err := d.saveCfg(); err != nil {
-		return err
-	}
-
-	return wrapnix.CreateWrappers(ctx, d)
+	return d.saveCfg()
```

Comment:
why don't we save the lockfile here, after the config has been changed? Similar to the end of AddPackages.

Reply:
It's actually not needed in either because `ensurePackagesAreInstalled` already does `lockfile.Tidy` 

will remove from `Add()`